src/styles/main.scss

```
@@ -0,0 +1,74 @@
+@mixin circle ($size) {
+  width: $size;
+  height: $size;
+  border-radius: 50%;
+}
+
+@mixin line ($length, $width) {
+  width: $length;
+  height: $width;
+  border-radius: $width / 2;
+}
+
+%Position {
```

be more specific with namings
```suggestion
%clock-hand-position {
```